pgjdbc/src/main/java/org/postgresql/ssl/LibPQFactory.java

```diff
@@ -217,4 +245,24 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
       }
     }
   }
+
+  private  X509CRL loadRevokedCertificates(CertificateFactory cf, String crlFile)
+      throws IOException, CRLException {
+
+    // load the CRL
+    X509CRL crl = null;
+    FileInputStream fis = null;
+    try {
+      new FileInputStream(crlFile);
```

Comment:
seems to be missing assigning to `fis`.

Reply:
amazingly it is ... thanks

Reply:
Is the existing test hitting this? As I read this method, it does not appear to ever actually load a revocation list.

Reply:
TBH, there will be a complete rewrite of this. Turns out CRL.isRevoked() use the CN as well as the serial number to see if it has been revoked. I am working on it now. I should probably close this PR